components/automate-chef-io/content/docs/client-runs.md

```diff
@@ -189,19 +205,20 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/missing-nodes/config
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or [use an Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests. 
+You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or [use an Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.

```

Comment:
```suggestion

```

Reply:
seems like we shouldnt be keeping this a1 content around, i added the corrected link to the line above

Reply:
I do not see a different link in your suggested change? I understand about the Chef Automate 1 removal. So this sentence needs to be modified to only mention creating a new token?

Reply:
this was being used, that does not work anymore
#creating-a-standard-api-token

this is the correct one
#creating-api-tokens

Reply:
I'm following more now. I was really thrown by the link not being included the correction, which you did in the other suggestions below. Incorporating